perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/PerunBlImpl.java

```diff
@@ -111,6 +113,11 @@
 	final static Logger log = LoggerFactory.getLogger(PerunBlImpl.class);
 
 	private final static Set<String> dontLookupUsersForLogins = BeansUtils.getCoreConfig().getDontLookupUsers();
+	private final static Set<String> extSourcesWithMultipleIdentifiers = BeansUtils.getCoreConfig().getExtSourcesMultipleIdentifiers();
+
+	private final static String multivalueAttributeSeparatorRegExp = ";";
+	private final static String additionalIdentifiersAttributeName = "additionalIdentifiers";
+	private final static String additionalIdentifiersPerunAttributeName = AttributesManager.NS_UES_ATTR_DEF + ":" + additionalIdentifiersAttributeName;
```

Comment:
These constants can be achieved from 1 shared place (like protected or public constants).